bfasst/utils/structural.py

```diff
@@ -22,11 +30,12 @@ class StructuralCompareError(Exception):
 class StructuralCompare:
     """Structural compare and map"""
 
-    def __init__(self, named_netlist_path, reversed_netlist_path, log_path) -> None:
+    def __init__(self, named_netlist_path, reversed_netlist_path, log_path, debug=False) -> None:
```

Comment:
Did you change other files to use this debug parameter? Or when you use the debug feature how are you setting it?

Reply:
I've been using a script that runs comparison directly without using ninja to allow me more flexibility and customization. Should I add it in as a flow argument?

Reply:
I think what we should do is have it dump the debug file by default, but only look for if the file exists if it is passed in as a command line argument. Maybe --debug should  tell it to look for the file, and it dumps the file regardless. How does that sound?

Reply:
That sounds good to me.